SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
-SKILL_20151224_005931
-SKILL_20151224_005932
-SKILL_20151224_005933
-SKILL_20151224_005934
-SKILL_20151224_005935
-SKILL_20151224_005936
-SKILL_20151224_005937
-SKILL_20151224_005938
-SKILL_20151224_005939
-SKILL_20151224_005940
-SKILL_20151224_005941
-SKILL_20151224_005942
-SKILL_20151224_005943
-SKILL_20151224_005944
-SKILL_20151224_005945
-SKILL_20151224_005946
-SKILL_20151224_005947
-SKILL_20151224_005948
-SKILL_20151224_005949
-SKILL_20151224_005950
-SKILL_20151224_005951
-SKILL_20151224_005952
-SKILL_20151224_005953
-SKILL_20151224_005954
-SKILL_20151224_005955
-SKILL_20151224_005956
-SKILL_20151224_005957
-SKILL_20151224_005958
-SKILL_20151224_005959
-SKILL_20151224_005960
-SKILL_20151224_005961
-SKILL_20151224_005962
-SKILL_20151224_005963
-SKILL_20151224_005964
-SKILL_20151224_005965
-SKILL_20151224_005966
-SKILL_20151224_005967
-SKILL_20151224_005968
-SKILL_20151224_005969
-SKILL_20151224_005970
-SKILL_20151224_005971
-SKILL_20151224_005972
-SKILL_20151224_005973
-SKILL_20151224_005974
-SKILL_20151224_005975
-SKILL_20151224_005976
-SKILL_20151224_005977
-SKILL_20151224_005978
-SKILL_20151224_005979
-SKILL_20151224_005980
-SKILL_20151224_005981
-SKILL_20151224_005982
-SKILL_20151224_005983
-SKILL_20151224_005984
-SKILL_20151224_005985
-SKILL_20151224_005986
-SKILL_20151224_005987
-SKILL_20151224_005988
-SKILL_20151224_005989
-SKILL_20151224_005990
-SKILL_20151224_005991
-SKILL_20151224_005992
-SKILL_20151224_005993
-SKILL_20151224_005994
-SKILL_20151224_005995
-SKILL_20151224_005996
-SKILL_20151224_005997
-SKILL_20151224_005998
-SKILL_20151224_005999
-SKILL_20151224_006000
-SKILL_20151224_006001
-SKILL_20151224_006002
-SKILL_20151224_006003
-SKILL_20151224_006004
-SKILL_20151224_006005
-SKILL_20151224_006006
-SKILL_20151224_006007
-SKILL_20151224_006008
-SKILL_20151224_006009
-SKILL_20151224_006010
-SKILL_20151224_006011
-SKILL_20151224_006012
-SKILL_20151224_006013
-SKILL_20151224_006014
-SKILL_20151224_006015
-SKILL_20151224_006016
-SKILL_20151224_006017
-SKILL_20151224_006018
-SKILL_20151224_006019
-SKILL_20151224_006020
-SKILL_20151224_006021
-SKILL_20151224_006022
-SKILL_20151224_006023
-SKILL_20151224_006024
-SKILL_20151224_006025
-SKILL_20151224_006026
-SKILL_20151224_006027
-SKILL_20151224_006028
-SKILL_20151224_006029
-SKILL_20151224_006030
-SKILL_20151224_006031
-SKILL_20151224_006032
+SKILL_20151224_005737	Mining
+SKILL_20151224_005738	Increase block count: #{CaptionRatio}#{nl}Attack reduction: #{CaptionTime}#%{nl}Critical hit rate reduction: #{CaptionRatio2}#{nl}Duration: 30 seconds
+SKILL_20151224_005739	Langort
+SKILL_20151224_005740	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Block and strike simultaneously by using both a sword and a shield.
+SKILL_20151224_005741	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Attack with descending vertical slash. Additional damage dealt onto defense reduced medium type and large type enemies from Scull Swing. Depending on the amount of harmful effect, additional damage is applied.
```

Comment:
Attack with a descending vertical slash.

Deals additional damage on medium and large-type enemies according to how much their defense has been decreased by Skull Swing.

An additional bonus damage will be applied depending on the amount of harmful effects that have been applied to the enemy.